imap_processing/glows/tests/test_glows_l1a_data.py

```diff
@@ -0,0 +1,220 @@
+import dataclasses
+from pathlib import Path
+
+import pytest
+
+from imap_processing.glows import version
+from imap_processing.glows.l0 import decom_glows
+from imap_processing.glows.l1.glows_l1a_data import DirectEventL1A, HistogramL1A
+from imap_processing.glows.utils.constants import DirectEvent, TimeTuple
+
+
+@pytest.fixture()
+def decom_test_data():
+    """Read test data from file"""
+    current_directory = Path(__file__).parent
+    packet_path = f"{current_directory}/glows_test_packet_20230920_v00.pkts"
```

Comment:
I think this will help our friends on Windows systems
```suggestion
    packet_path = current_directory / "glows_test_packet_20230920_v00.pkts"
```

Reply:
Every time 😆 why does windows do this to me?!